docs/content/developers/index.md

```diff
@@ -0,0 +1,79 @@
+# Technische documentatie voor developers
+
+De standaard wordt opgebouwd op basis van drie componenten, elk met hun eigen
+OAS 3.0 definities. Het gaat dan om de zaakregistratiecomponent (ZRC),
+documentregistratiecomponent (DRC) en de zaaktypecatalogus (ZTC).
+
+De ontwikkeling van iedere component vindt plaats in een aparte git repository:
+
+* [ZRC](https://github.com/vng-Realisatie/gemma-zaakregistratiecomponent)
+* [DRC](https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent)
+* [ZTC](https://github.com/VNG-Realisatie/gemma-zaaktypecatalogus)
+
+Uitgebreide documentatie per component is beschikbaar in deze repositories.
+Dit document is als startpunt bedoeld om snel up-and-running te zijn.
+
+## Ontwikkelen van een applicatie die van deze componenten gebruik maakt
+
+Bij het ontwikkelen van vakapplicaties (ook wel afhandelapplicaties genoemd),
+kunnen deze referentie-implementaties ingezet worden als backend services.
+
+### Services starten
+
+Alle componenten zijn als Docker containers beschikbaar.
+
+In `infra` staat een bruikbare `docker-compose.yml` docker-compose file.
+Hiervoor heb je nodig:
+
+* Docker
+* Docker-compose
+
+Voorbeeld gebruik:
+
+```bash
+cd infra
+docker-compose pull  # update naar nieuwste images
+docker-compose up -d
+```
+
+Je kan nu de componenten bereiken:
+
+* ZRC: http://localhost:8000
+* DRC: http://localhost:8001
```

Comment:
Wanneer je landt op drc zie je in beeld "Overigedataregistratiecomponent", maar het is de verwachtte DRC referentieimplementatie. Zou netjes zijn dat nog te wijzigen.

Reply:
Verplaatst naar https://github.com/VNG-Realisatie/gemma-documentregistratiecomponent/issues/9 , staat los van werking.